media-libs/kimageannotator/kimageannotator-0.6.0.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 2020-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+VIRTUALX_REQUIRED="test"
+inherit cmake virtualx
+
+MY_PN=kImageAnnotator
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="Tool for annotating images"
+HOMEPAGE="https://github.com/ksnip/kImageAnnotator"
+SRC_URI="https://github.com/ksnip/${MY_PN}/archive/v${PV}.tar.gz -> ${MY_P}.tar.gz"
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="LGPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~ppc64 ~riscv ~x86"
+IUSE="test"
+
+RDEPEND="
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5
+	dev-qt/qtsvg:5
+	dev-qt/qtwidgets:5[png]
+	>=media-libs/kcolorpicker-0.2.0
+	x11-libs/libX11
+"
+DEPEND="${RDEPEND}
+	test? (
+			dev-qt/qttest:5
+			dev-cpp/gtest
+	)
+"
+BDEPEND="
+	dev-qt/linguist-tools:5
+"
+
+src_configure() {
+	local mycmakeargs=(
+		-DBUILD_TESTS=$(usex test)
+	)
+	cmake_src_configure
+}
+
+src_test() {
+	BUILD_DIR="${BUILD_DIR}/tests" virtx cmake_src_test
```

Comment:
This seems okay, although I wonder if you can do a `cd` instead.

Reply:
Unless I misunderstand, I just tried this and it still did not run tests:

```
src_test() {
        cd ${BUILD_DIR}/tests
        virtx cmake_src_test
}
```

Reply:
Something happens here with your current implementation,
```
100% tests passed, 0 tests failed out of 49
```